pom.xml

```diff
@@ -256,190 +254,197 @@
 			<artifactId>jackson-core</artifactId>
 			<version>${jackson.version}</version>
 		</dependency>
-
+		  
+		<!-- JSQLFormatter -->
```

Comment:
Whatever you did, it seems it made more changes than necessary, but it's also fine, no worries, I'll handle it 😄